### ISSUE TITLE
After moving the sign up sheet over one week, add protections to the row with the corresponding date

### ISSUE THREAD

**Issue by bshaibu** [Mountain-View-Toastmasters/sign-up-sheet#12]
We should lock cells that have already been edited to only be modified by officers so there are no (well fewer) accidental overwrites. This should only happen at the end of the Advance Sign Up Sheet script when that week's row is "complete".

**Comment by bshaibu** [comment]
And to detail my blunder,

I set the entire [Roles sheet](https://docs.google.com/spreadsheets/d/1edC7fSgwzDEALY_RZa3rnV80KVpoGGr9WCb6chnjxnE/edit#gid=1744068420) to protected instead of just a range.
My goal was to not have to repeatedly change what rows are marked as protected each week.

This backfired because only the officers list and a couple others were in the protected range. When the scripts run they use permissions for the current user - who is likely not an officer.
To fix this, I just removed the protection from the entire sheet and protected only the rows before the current week's row. The current week (and following weeks) will need to be protected once they are "complete".

TL;DR I tried to be clever and lazy and it backfired so we'll have to programmatically protect new rows like we discussed way back.
